cluster/test-smoke.sh

```diff
@@ -23,10 +23,10 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 
-TEST_ARGS="$@"
+TEST_ARGS="$*"
```

Comment:
```
In ./cluster/test-smoke.sh line 28:
TEST_ARGS="$@"
          ^-- SC2124: Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.
```